community/urls.py

```diff
@@ -37,6 +38,12 @@ def get_index():
         distill_func=get_index,
         distill_file='activity/index.html',
     ),
+    distill_url(
+        r'gci/rss/', gci_rss(),
+        name='gci-rss',
+        distill_func=get_index,
+        distill_file='gci/rss/index.xml',
```

Comment:
`gci/tasks/rss.xml` ?

Reply:
`index.xml` is better since then you don't have to type the whole thing, how about `gci/tasks-rss/` or something then?